<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CamerasDialog</class>
 <widget class="QDialog" name="CamerasDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>198</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CamerasDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="addCameraButton">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeCamerasButton">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="cameraList">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="OKButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>OKButton</sender>
   <signal>clicked()</signal>
   <receiver>CamerasDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
